jwst/cube_build/ifu_cube.py

```diff
@@ -315,13 +321,14 @@ def set_geometry(self, footprint):
             self.lambda_max = self.lambda_min + (self.naxis3) * self.cdelt3
 
             self.zcoord = np.zeros(self.naxis3)
-            self.crval3 = self.lambda_min
+            # CRPIX3 for FITS is 1 (center of first pixel)
+            # CRVAL3 then is lambda_min + self.cdelt3/ 2.0, which is also zcoord[0]
```

Comment:
So does lambda_min correspond to the left edge of the first pixel, and hence you have to add cdelt3/2 to get it to the center of the pixel?

Reply:
yes